.github/workflows/ci.yml

```diff
@@ -1,38 +1,26 @@
 name: CI
 
 on:
+  workflow_dispatch:
```

Comment:
What does this achieve?  I haven't worked with it before.

Reply:
It allows me to manually trigger a CI build on my repo, where not every branch is built by default. Not really needed for this repo here though